Softeq.XToolkit.WhiteLabel.iOS/AppDelegateBase.cs

```diff
@@ -35,7 +36,16 @@ protected virtual void InitializeWhiteLabelRuntime()
             PlatformProvider.Current = new IosPlatformProvider();
 
             // Init dependencies
-            Bootstrapper.Initialize();
+            var bootstrapper = CreateBootstrapper();
+            var container = bootstrapper.Initialize();
+            Dependencies.Initialize(container);
```

Comment:
Good for Bootstrapper, but can be not so obvious at all for infrastructure.

Reply:
To my understanding, the only responsibility of Bootstrapper is to configure IoC container (now it should be obvious just by looking into IBootstrapper interface). So I moved registration of the container in the app infrastructure outside - to AppDelegete and MainApplication respectively. There's even a method there `InitializeWhiteLabelRuntime()` - so AppDelegete and MainApplication are really responsible for initializing a runtime, and IoC container is just a part of it.
And one last point: AppDelegete and MainApplication expect an implementation of IBootstrapper interface, theoretically it could be any implementation, and it is not obvious that this implementation **must** call `Dependencies.Initialize(container);`

Reply:
Cool, I agree.